django_webapps/scripts/db_population/parsers/teacher_parsers.py

```diff
@@ -14,88 +12,52 @@ class ProjectStatusAction(str, Enum):
     NONE = "none"
 
 
-def parse_project_status(
+RAW_PROJECT_STATUS_TO_PARSED: dict[
+    str, tuple[TeacherProjectStatus | TeacherSituationalStatus, ProjectStatusAction]
+] = {
+    "идут занятия": (TeacherProjectStatus.WORKING, ProjectStatusAction.NONE),
+    "перерыв в преподавании": (TeacherProjectStatus.ON_LEAVE, ProjectStatusAction.NONE),
+    "закончил участие в проекте": (TeacherProjectStatus.FINISHED_LEFT, ProjectStatusAction.NONE),
```

Comment:
just checking: we know that these teachers will not be in the new database but this low-level module doesn't have to (or shouldn't) know about it, so we assign a status here anyway?

Reply:
Yeh, kind of but then we need to create some intermediate data structure, I think it is safe to assume that choice is low-level enough